KarhooUISDK/Translations/de.lproj/Localizable.strings

```diff
@@ -141,6 +141,11 @@
 
 "kh_uisdk_accessibility_label_train_number" = "Zugnummer";
 
+"kh_uisdk_accessibility_loyalty_switch_disabled" = "";
```

Comment:
Texts not translated. Auto translation should be good enough until they are validated. This way we get something for other languages than English. Otherwise we need to not forget to do it before a release.